src/core/utils.js

```diff
@@ -8,6 +8,20 @@ import { pub } from "./pubsubhub.js";
 export const name = "core/utils";
 
 const dashes = /-/g;
+/**
+ * Hashes a string from char code.
+ * https://stackoverflow.com/questions/7616461/generate-a-hash-from-string-in-javascript
+ *
+ * @param {String} text
+ */
+function hashString(text) {
```

Comment:
This can return negative numbers also. Not a problem, but a note.

Reply:
Added note.